### PR TITLE
Migration of Tex

### PR DESCRIPTION
**Pull request by DomT4**
Part 1 of #38819.

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Before we move these it may be worth investigating if any of them can be made optional e.g. `lilypond`

**Comment by DomT4** [comment]
Wilco :+1:

**Comment by DomT4** [comment]
Updating as I move through:

* auctex - Breaks on compile, no configure option to opt-out.
* chktex - Breaks on compile, no configure opt-out.
* latex-mk - Breaks on compile, no configure opt-out.
* latex2html - Works without ` tex `. Can be made optional.

**Comment by DomT4** [comment]
Pushed updated commit.

File: Library/Homebrew/tap_migrations.rb [hunk 1]
```diff
@@ -96,10 +98,13 @@
   "kernagic" => "homebrew/x11",
   "kismet" => "homebrew/boneyard",
   "klavaro" => "homebrew/x11",
+  "latex-mk" => "homebrew/tex",
+  "latex2html" => "homebrew/tex",
```
Comment (mikemcquaid) [inline Library/Homebrew/tap_migrations.rb]: This needs :skull:ed I think.

Comment (DomT4) [inline Library/Homebrew/tap_migrations.rb]: Whoops! Nice spot. Will fix now.

**Comment by DomT4** [comment]
@mikemcquaid I removed ` lilypond ` from this PR for now. Everything else is successfully merged into homebrew/tex. Can be merged once the bot is green and you're happy. Will keep working on making the bot more verbose so Travis doesn't freak on the longer builds like Lilypond.

**Comment by mikemcquaid** [comment]
Thanks @DomT4!

**Pull request closed without merging**